app/background/wallet/service.js

```diff
@@ -547,6 +550,7 @@ class WalletService {
       prefix: networkName === 'main'
         ? HSD_DATA_DIR
         : path.join(HSD_DATA_DIR, networkName),
+      migrate: 0,
```

Comment:
Won't this prevent any future wallet migrations from going through?

Reply:
For more context here is the original [PR](https://github.com/kyokan/bob-wallet/pull/200) and intent before the `migrate` flag addition. It is for an existing migration in `hsd`.